docs/blog/newsletter-august-2019.rst

```diff
@@ -22,17 +20,17 @@ Adventures in generating docs from code comments
 
 * Javadoc is pretty standard in the Java world, so you can expect that Java devs are already looking for the Javadoc - and you can help them just by making sure it's easy to find.
-* No matter which tool you use, the content is only as good as the comments in the code. Great documentation typically depends on collaboration between developers and writers, which documentarians agreed can be challenging. To improve developer buy-in, try emphasizing that better docs (through more complete and accurate code comments) can help increase adoption: no one will use the product if they can't understand it. Any tactic to open a conversation can help: one writer started by contributing basic grammar edits.
+* No matter which tool you use, the content is only as good as the comments in the code. Great documentation depends on collaboration between developers and writers, which can be challenging. To improve developer buy-in, try emphasizing that more complete and accurate code comments can help increase adoption: no one will use the product if they can't understand it. Any tactic to open a conversation can help: one writer started by contributing basic grammar edits.
 * When it comes to what the docs *look* like, you can customize the standard doclet yourself or search for existing custom doclets. Take care with custom doclets that aren't maintained or supported: they can take a lot of work. And remember, Javadoc and Doxygen actually only "autogenerate" insofar as they are integrated into the app build process.
-* Javadoc interprets the code itself, so it can - even without code comments - generate basic docs on package, class, method, field, and argument names and types. One person mentioned they sometimes run Javadoc just to see what it turns up about the code that isn't documented elsewhere. Because Javadoc includes fields for doc elements, you can see which are blank when you generate the docs.
+* Javadoc interprets the code itself, so it can - even without code comments - generate basic docs on package, class, method, field, and argument names and types. One person mentioned they sometimes run Javadoc just to see what it turns up about the code that isn't documented elsewhere.
 
-For more about a successful Javadoc/doxygen project that included writers and developers, check out Jennifer Rondeau's `Symantec case study <https://www.docslikecode.com/articles/symantec-case-study/>`__. For a discussion of the benefits of writers and developers working together, see Beth Aitman's talk `Who Writes the Docs? </videos/portland/2018/who-writes-the-docs-beth-aitman/>`__.
+For more about a successful Javadoc/doxygen project that included writers and developers, check out Jennifer Rondeau's `Symantec case study <https://www.docslikecode.com/articles/symantec-case-study/>`__. For a discussion of the benefits of writers and developers working together, see our editor's talk `Who Writes the Docs? </videos/portland/2018/who-writes-the-docs-beth-aitman/>`__.
```

Comment:
I didn't notice this until I looked at the preview, but the period after the link setup for Who Writes the Docs? should be removed:
...talk `Who Writes the Docs? </videos/portland/2018/who-writes-the-docs-beth-aitman/>`__

Reply:
Good spot, thank you!